src/components/about-project/sections/Timeline.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { createStyles, Grid, makeStyles, Theme, Typography } from '@material-ui/core'
+import { Timeline as TimelineMaterial } from '@material-ui/lab'
+import PlayCircleFilledWhiteIcon from '@material-ui/icons/PlayCircleFilledWhite'
+import CodeIcon from '@material-ui/icons/Code'
+import TelegramIcon from '@material-ui/icons/Telegram'
+import UpdateIcon from '@material-ui/icons/Update'
+
+import TimelineItem from './TimelineItem'
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    heading: {
+      marginBottom: theme.spacing(5),
+      color: theme.palette.primary.dark,
+      fontSize: theme.spacing(5),
+    },
+    container: {
+      marginBottom: theme.spacing(12),
+      textAlign: 'center',
+    },
+    content: {
+      fontSize: theme.spacing(2.5),
+      maxWidth: theme.spacing(20),
+    },
+  }),
+)
+
+export default function Timeline() {
+  const { t } = useTranslation()
+  const classes = useStyles()
+
+  return (
+    <Grid container direction="column" component="section" className={classes.container}>
+      <Typography variant="h4" component="h2" className={classes.heading}>
+        {t('about-project:timeline')}
+      </Typography>
+      <Grid item>
+        <TimelineMaterial align="alternate">
+          <TimelineItem Icon={CodeIcon}>
```

Comment:
@uzuntonev Maybe for the next revision we can extract the timeline items as external const array as it will make extending of the list easier